fiori/app/orders/webapp/Component.js

```diff
@@ -1,5 +1,8 @@
-sap.ui.define(["sap/fe/core/AppComponent"], ac => ac.extend("orders.Component", {
-    metadata:{ manifest:'json' }
-}))
+sap.ui.define(["sap/fe/core/AppComponent"], function(AppComponent) {
```

Comment:
Wonder what the real difference is.  Isn't this just Javascript syntax change?

Reply:
Arrow function is not supported on IE11 so should not be used within UI5 :)